Tasks/AzureTestPlanV0/constants.ts

```diff
@@ -7,3 +7,5 @@ export const AUTOMATED_TEST_NAME = "Microsoft.VSTS.TCM.AutomatedTestName";
 export const AUTOMATED_TEST_STORAGE = "Microsoft.VSTS.TCM.AutomatedTestStorage";
 export const AUTOMATION_STATUS = 'Microsoft.VSTS.TCM.AutomationStatus';
 export const NOT_AUTOMATED = 'Not Automated'
+export const MERGE_THRESHOLD = 100;
+export const TESTRUN_SYSTEM = 'VSTS - ATP';
```

Comment:
we need to also add the framework like is it Maven or Graddle or Python. Please seed this accordingly.

Reply:
Updated